### ISSUE TITLE
Error: Listing nft when using openseaSDk and metamask (version: >6.1.8)

### ISSUE THREAD

**Issue by lehongvo** [ProjectOpenSea/opensea-js#1221]
### Component

API

### Have you ensured that all of these are up to date?

- [X] opensea-js
- [X] Node (minimum v16)

### What version of opensea-js are you on?

 >6.1.8

### What function is the bug in?

createSellOrder

### Operating System

macOS (Apple Silicon)

### Describe the bug

**Follow link: https://github.com/ProjectOpenSea/opensea-js/discussions/1219**
Source code: 
```
order = await openSeaSdk.createSellOrder({
					asset: {
						tokenAddress: tokenAddress,
						tokenId: tokenId,
						tokenStandard: constants.TOKEN_STANDARD
					},
					accountAddress: accountAddress,
					startAmount: startAmount,
					expirationTime: expirationTime
				})

```

Package.json
```
{
  "name": "ntf-marketplace-fe",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev -p 3000",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "format": "prettier --write 'src/*.ts' 'src/**/*.ts'",
    "genContractType": "abi-types-generator './public/contracts/NftMarket.json' --output='./types' --name=nftMarketContract --provider=ethers_v5"
  },
  "dependencies": {
    "@ant-design/charts": "^1.4.2",
    "@ant-design/cssinjs": "^1.16.0",
    "@ant-design/plots": "^1.2.5",
    "@metamask/providers": "^11.1.1",
    "@openzeppelin/contracts": "^4.9.3",
    "@reduxjs/toolkit": "^1.9.5",
    "@truffle/hdwallet-provider": "^2.1.13",
    "@types/axios": "^0.14.0",
    "@types/node": "20.4.5",
    "@types/react": "18.2.17",
    "@types/react-dom": "18.2.7",
    "@types/react-icons": "^3.0.0",
    "@types/react-infinite-scroll-component": "^5.0.0",
    "@types/redux-saga": "^0.10.5",
    "antd": "^5.7.3",
    "axios": "^1.4.0",
    "eslint": "8.45.0",
    "eslint-config-next": "13.4.12",
    "ethers": "5.7.2",
    "next": "13.4.12",
    "opensea-js": "^6.1.8",
    "prettier": "^3.0.3",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "react-icons": "^4.11.0",
    "react-infinite-scroll-component": "^6.1.0",
    "react-redux": "^8.1.2",
    "redux-saga": "^1.2.3",
    "styled-components": "^6.0.5",
    "swr": "^2.2.2",
    "typescript": "5.1.6"
  },
  "devDependencies": {
    "autoprefixer": "^10.4.14",
    "ethereum-abi-types-generator": "^1.3.4",
    "postcss": "^8.4.27",
    "tailwindcss": "^3.3.3"
  }
}

```

**Comment by ryanio** [comment]
How are you initializing `openSeaSdk`, please share a complete code example. You should pass `window.ethereum` provider and call `eth_requestAccounts` via the provider to get access to metamask account.

**Comment by ryanio** [comment]
Also, what error are you receiving?

**Comment by lehongvo** [comment]
Also, what error are you receiving?
===> Error:
Error: Specified accountAddress is not available through wallet or provider: 0x76C4ad7477C253aeFBC395eE20dad91caE0Be899(==> My wallet)

**Comment by lehongvo** [comment]
Source code: initializing openSeaSdk
```
export const useOpenseaSdk = () => {
	const { hooks, provider } = useHooks(); // Prodider get from metamask
	const { data } = hooks.useNetwork();
	const [openSeaSdk, setOpenSeaSdk] = useState<OpenSeaSDK | null>(null);
	useMemo(() => {
		if (provider && data && data !== 'Unknown Network') {
			const chains = Object.values(Chain);
			const chainStr = chains.find((chain) => chain === data);
			const openseaSdk = new OpenSeaSDK(provider!, {
				chain: chainStr, // Chain.Mumbai
			});
			setOpenSeaSdk(openseaSdk);
		}
	}, [data, provider]);

	return {
		openSeaSdk
	};
};
```

**Comment by lehongvo** [comment]
@ryanio Please check issue. Thank you so much

**Comment by ryanio** [comment]
This is now released in [v6.1.11](https://github.com/ProjectOpenSea/opensea-js/releases/tag/v6.1.11), please give it a try and let us know if it works for you.
